### PR TITLE
feat: add accept term and privacy query

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

**Review by davidsoderberg — APPROVED** [review]
Nice 👍

File: packages/cli/src/commands/init.consts.ts [hunk 1]
```diff
@@ -67,6 +67,24 @@ export const registerMethodQuestions: ListQuestionOptions[] = [
   },
 ];
 
+export const termAndPrivacyQuestions: ListQuestionOptions[] = [
+  {
+    type: 'list',
+    name: 'accept',
+    message: `I accept the https://novu.co/terms and have read the  https://novu.co/privacy`,
```
[the remark this answers is not in the document]
Comment (djabarovgeorge) [reply packages/cli/src/commands/init.consts.ts:74]: Validate if we can make the links as hypertext

Comment (scopsy) [inline packages/cli/src/commands/init.consts.ts:74]:
```suggestion
    message: `I accept the Terms and Conditions (https://novu.co/terms) and have read the Privacy Policy (https://novu.co/privacy)`,
```

**Review by scopsy — COMMENTED** [review]
Awesome! @djabarovgeorge could you maybe create a short loom/gif for the flow?

**Comment by djabarovgeorge** [comment]
> Awesome! @djabarovgeorge could you maybe create a short loom/gif for the flow?

Sure, 
https://www.loom.com/share/e45c8971e13d4d55b5405fb9c9de81f4

**Pull request merged**